### PR TITLE
etcdserver: collect error from errorc

### LINKED ISSUE #1993: move the output of the removal msgs to one place
currently etcd also prints out removal msgs in sender.go.
we need to move it to server.go.

### PR DESCRIPTION
**Pull request by xiang90**
Fix #1993

### CODE REVIEW

File: etcdserver/server.go [hunk 1]
```diff
@@ -420,9 +421,7 @@ func (s *EtcdServer) run() {
 				}
 				if len(ents) > 0 {
 					if appliedi, shouldstop = s.apply(ents, &confState); shouldstop {
-						m1 := fmt.Sprintf("etcdserver: removed local member %s from cluster %s", s.ID(), s.Cluster.ID())
-						m2 := fmt.Sprint("etcdserver: the data-dir used by this member must be removed so that this host can be re-added with a new member ID")
-						go s.stopWithDelay(10*100*time.Millisecond, m1, m2)
+						go s.stopWithDelay(10*100*time.Millisecond, fmt.Errorf("the member has been permanently removed from the cluster"))
```
Comment (yichengq) [inline etcdserver/server.go]: print out member ID and cluster ID too as in https://github.com/coreos/etcd/pull/2035/files#diff-971f6af906b31cc86efd71aff56aa96bL423 ?

**Comment by yichengq** [comment]
lgtm after addressing the comment

File: etcdserver/server.go (hunk 1 above)
Comment (xiang90) [inline etcdserver/server.go]: i think it is verbose and gives on valuable information in this case.

Comment (yichengq) [inline etcdserver/server.go]: kk

**Pull request merged**